tests/verifyPMMSettingsPageFunctionality_test.js

```diff
@@ -148,18 +144,11 @@ Scenario(
     // Verify tooltips for Communication > Email fields
     for (const o of Object.keys(pmmSettingsPage.communication.email)) {
```

Comment:
I'd rather use better naming for variable, not o, use 
`const field of Object.keys..`

Reply:
refactored